### PR TITLE
Study overview: adding field - created by

### PR DESCRIPTION
**Pull request by davidscavnicky**
Adding createdBy field to studyOverview

### CODE REVIEW

File: src/main/kotlin/dk/cachet/carp/webservices/study/repository/CoreStudyRepository.kt [hunk 1]
```diff
@@ -99,6 +99,7 @@ class CoreStudyRepository
                     studyStatus.studyId,
                     studyStatus.name,
                     studyStatus.createdOn,
+                    UUID(it.createdBy.toString()),
```
Comment (xelahalo) [inline src/main/kotlin/dk/cachet/carp/webservices/study/repository/CoreStudyRepository.kt:102]: if you call toString on a nullable property it is just going to be `"null"`, which is going to error out when you try to instantiate a `UUID`

Comment (xelahalo) [reply src/main/kotlin/dk/cachet/carp/webservices/study/repository/CoreStudyRepository.kt:102]: Instead, try to check if it exists in the first place and handle the request accordingly

Comment (davidscavnicky) [reply src/main/kotlin/dk/cachet/carp/webservices/study/repository/CoreStudyRepository.kt:102]: applying UUID(createdBy) auto checks if it exists in the first place if it is not the correct string format for UUID or its null; good enough;

**Comment by davidscavnicky** [comment]
for the case if string createdBy is null(not exist) or not UUID() case, it returns null for createdBy

**Comment by davidscavnicky** [comment]
@xelahalo

**Pull request closed without merging**